core-services/image-mirroring/hypershift/mapping_hypershift_4_12

```diff
@@ -1,2 +1,2 @@
-registry.ci.openshift.org/ocp/4.12:hypershift quay.io/hypershift/hypershift:4.12 quay.io/hypershift/hypershift:latest
-registry.ci.openshift.org/ocp/4.12:hypershift-operator quay.io/hypershift/hypershift-operator:4.12  quay.io/hypershift/hypershift-operator:latest
+registry.ci.openshift.org/ocp/4.12:hypershift quay.io/hypershift/hypershift:4.12 quay.io/hypershift/hypershift:4.12
```

Comment:
we just need to remove this last entry (instead of changing 4.12 to latest)
same below